source/_components/switch.aqualogic.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "AquaLogic Switch"
+description: "Instructions on how to integrate your AquaLogic devices within Home Assistant."
+date: 2018-09-17 9:16
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hayward.png
+ha_category: Switch
+ha_release: "0.80"
+ha_iot_class: "Local Push"
+---
+
+To get your AquaLogic switches working within Home Assistant, please follow the instructions for the general [AquaLogic component](/components/aqualogic).
+
+## {% linkable_title Configuration %}
+
+Once you have enabled the [AquaLogic component](/components/aqualogic), add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: aqualogic
+    monitored_conditions:
+      - lights
+      - filter
+```
+
+{% configuration %}
+monitored_conditions:
+  description: List of items you want to monitor/control.
+  required: true
```

Comment:
This isn't required according to the source. Since there's a default it doesn't matter too much though.